content/en/integrations/guide/source-code-integration.md

```diff
@@ -526,9 +506,7 @@ You can see links from errors in your Lambda functions' associated stack traces
 
 1. Navigate to [**Infrastructure** > **Serverless**][101] and click on the **AWS** tab.
 2. Click on a Lambda function and click the **Open Trace** button for an invocation with an associated stack trace.
-3. Click **View Code** to open the error in its source code repository.
-
-If you're using the GitHub integration, click **Connect to preview** on error frames. You can see inline code snippets directly in the Lambda function's stack trace.
+3. If you're using the GitHub or GitLab integrations, click **Connect to preview** on stack frames. You can see inline code snippets directly in the stack trace. Otherwise, you can always click the **View** button on the right of a frame or select **View file**, **View Git blame**, or **View commit** to be redirected to your source code management tool.
```

Comment:
```suggestion
3. If you're using the GitHub or GitLab integrations, click **Connect to preview** on stack frames. You can see inline code snippets directly in the stack trace. Otherwise, you can click the **View** button on the right of a frame or select **View file**, **View Git blame**, or **View commit** to be redirected to your source code management tool.
```